src/node_v8.cc

```diff
@@ -20,25 +20,37 @@ using v8::Uint32;
 using v8::V8;
 using v8::Value;
 
+#define HEAP_STATISTICS_PROPERTIES(V)                                         \
+  V(0, total_heap_size, kTotalHeapSizeIndex);                                 \
+  V(1, total_heap_size_executable, kTotalHeapSizeExecutableIndex);            \
+  V(2, total_physical_size, kTotalPhysicalSizeIndex);                         \
+  V(3, used_heap_size, kUsedHeapSizeIndex);                                   \
+  V(4, heap_size_limit, kHeapSizeLimitIndex);
```

Comment:
Sorry to go on about this but please don't use semicolons in x-macros, they tend to make code and error messages very confusing.